requirements.txt

```diff
@@ -11,3 +11,4 @@ SQLAlchemy
 wtforms
 dogpile.cache
 blinker # required by Flask for the unit-tests (for signaling)
+nose===1.3.0
```

Comment:
That should be in a test_requirements.txt and I very much doubt we can't work with another version than this very specific one :)